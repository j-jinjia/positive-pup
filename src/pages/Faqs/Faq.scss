.faq-section {
  margin: 4rem;
}
