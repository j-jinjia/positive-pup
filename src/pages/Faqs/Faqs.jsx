import Dropdown from "../../components/Dropdown/Dropdown";
import Header from "../../components/Header/Header";
import Layout from "../../components/Layout/Layout";
import faqsData from "../../assets/mockData/faqsData";

const Faqs = () => {
  const faqSection = faqsData.map((data) => (
    <Dropdown key={data.id} question={data.question} answer={data.answer} />
  ));

  return (
    <Layout>
      <h1>FAQs</h1>
      <Header
        isLeftAlign={false}
        headingText={"Frequently Asked Questions"}
        subheadingText={"Some of your most common questions"}
        isAlternative={true}
      />
      {faqSection}
    </Layout>
  );
};

export default Faqs;
