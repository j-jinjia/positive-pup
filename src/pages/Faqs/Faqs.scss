@use "../../styles/variables" as *;

.faqs {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100%;
  margin: 4rem 0;
}
.header {
  
  &__heading{
    font-size: $font-size-large;
    margin-top: 3rem;
  }

  &__subheading{
    margin-bottom: 4.3rem;
  }
}

@media screen and (min-width: $tablet-breakpoint) {
.header {
  &__heading{
    font-size: $font-size-larger;
  }
  &--alternative{
    margin-bottom: 6.25rem;
    padding-top: 0;
  }
}
}