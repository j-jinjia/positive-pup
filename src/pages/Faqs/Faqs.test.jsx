import { customRender } from "../../utils/testUtils";
import Faqs from "./Faqs";

it("should render the Faqs page", () => {
  const { container } = customRender(<Faqs />);

  expect(container).toMatchSnapshot();
});
