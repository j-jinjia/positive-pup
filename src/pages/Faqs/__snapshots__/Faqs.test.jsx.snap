// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Faqs page 1`] = `
<div>
  <div
    class="layout"
  >
    <div
      class="navbar"
    >
      <div
        class="navbar__container"
      >
        <a
          href="/"
        >
          <img
            alt="The Positive Pup logo"
            class="navbar__container-logo"
            src="navbar-logo.png"
          />
        </a>
        <nav
          class="navbar__nav"
        >
          <a
            class="navbar__nav-item"
            href="/about"
          >
            About
          </a>
          <button
            class="navbar__nav-item navbar__nav-item--button"
          >
            Services
            <img
              class="navbar__nav-item-arrow"
              src="dropdown-arrow.svg"
            />
          </button>
          <a
            class="navbar__nav-item"
            href="/testimonials"
          >
            Testimonials
          </a>
          <a
            class="navbar__nav-item"
            href="/gallery"
          >
            Gallery
          </a>
          <span
            class="navbar__nav-item navbar__nav-item--phone"
          >
            <img
              alt=""
              class="navbar__nav-item-button"
              src="phone.svg"
            />
            <a
              class="navbar__nav-item"
              href="/get-in-touch"
            >
              Get in Touch
            </a>
          </span>
          <div
            class="button button--primary button--link"
          >
            <a
              href="/book-now"
            >
              Book Now
            </a>
          </div>
        </nav>
      </div>
    </div>
    <div
      class="header header--alternative"
    >
      <h1
        class="header__heading"
      >
         
        Frequently Asked Questions
         
      </h1>
      <h2
        class="header__subheading"
      >
        Some of your most common questions
      </h2>
    </div>
    <div
      class="category-filter category-filter--desktop category-filter--alternative-highlight"
    >
      <button
        class="category-filter__option category-filter__option--selected"
        value="General"
      >
        General
      </button>
      <button
        class="category-filter__option"
        value="Payments"
      >
        Payments
      </button>
      <button
        class="category-filter__option"
        value="Training Methods"
      >
        Training Methods
      </button>
      <button
        class="category-filter__option"
        value="Packages"
      >
        Packages
      </button>
      <button
        class="category-filter__option"
        value="Group Classes"
      >
        Group Classes
      </button>
    </div>
    <div
      class="faqs"
    >
      <div
        class="dropdown"
      >
        <label
          class="dropdown__heading-label"
          for="dropdown"
        >
          Click to view answer to What are the payments methods that you take?
        </label>
        <button
          class="dropdown__heading"
          id="dropdown"
        >
          <h3
            class="dropdown__heading-title"
          >
            What are the payments methods that you take?
          </h3>
          <p
            class="dropdown__heading-action"
          >
            +
          </p>
        </button>
        <div
          class="dropdown__main"
        >
          <p
            class="dropdown__main-text"
          >
            That is a great question! Finding the right trainer isn’t easy and can be a scary prospect.  The last thing you want to do is waste money on the wrong solution and cause any long term problems for your dog, right?
          </p>
          <br />
          <p
            class="dropdown__main-text"
          >
            Because of this it will be reassuring to you to know that during my time at the Victoria Stilwell Academy, I have had the benefit of training from some of the best people within the industry. Even more encouraging is that the positive reinforcement method I have been trained in is backed heaps of by scientific evidence and is proven to be the best method available to achieve outstanding results.
          </p>
          <br />
          <p
            class="dropdown__main-text"
          >
            To add to this you can guarantee that your dog will enjoy these sessions and will love to learn!
          </p>
        </div>
      </div>
      <div
        class="dropdown"
      >
        <label
          class="dropdown__heading-label"
          for="dropdown"
        >
          Click to view answer to What training methods do you use?
        </label>
        <button
          class="dropdown__heading"
          id="dropdown"
        >
          <h3
            class="dropdown__heading-title"
          >
            What training methods do you use?
          </h3>
          <p
            class="dropdown__heading-action"
          >
            +
          </p>
        </button>
        <div
          class="dropdown__main"
        >
          <p
            class="dropdown__main-text"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
          </p>
        </div>
      </div>
      <div
        class="dropdown"
      >
        <label
          class="dropdown__heading-label"
          for="dropdown"
        >
          Click to view answer to Where are your classes held?
        </label>
        <button
          class="dropdown__heading"
          id="dropdown"
        >
          <h3
            class="dropdown__heading-title"
          >
            Where are your classes held?
          </h3>
          <p
            class="dropdown__heading-action"
          >
            +
          </p>
        </button>
        <div
          class="dropdown__main"
        >
          <p
            class="dropdown__main-text"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
          </p>
        </div>
      </div>
      <div
        class="dropdown"
      >
        <label
          class="dropdown__heading-label"
          for="dropdown"
        >
          Click to view answer to How big are your group classes?
        </label>
        <button
          class="dropdown__heading"
          id="dropdown"
        >
          <h3
            class="dropdown__heading-title"
          >
            How big are your group classes?
          </h3>
          <p
            class="dropdown__heading-action"
          >
            +
          </p>
        </button>
        <div
          class="dropdown__main"
        >
          <p
            class="dropdown__main-text"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
          </p>
        </div>
      </div>
    </div>
    <div
      class="faq-get-in-touch"
    >
      <div
        class="header"
      >
        <h2
          class="header__subheading"
        >
          Can't find an answer to your Question?
        </h2>
      </div>
      <p
        class="faq-get-in-touch__paragraph"
      >
        Please get in touch below if you can’t find an answer you’re looking for. I’d love to hear from you! Thanks Mel
      </p>
      <div
        class="button button--secondary button--link"
      >
        <a
          href="/get-in-touch"
        >
          GET IN TOUCH
        </a>
      </div>
    </div>
    <div
      class="footer"
    >
      <div
        class="footer__image-container"
      >
        <img
          alt="logo"
          class="footer__logo"
          src="footer-logo.svg"
        />
      </div>
      <ul
        class="footer__links-container"
      >
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/about"
          >
            About
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/services"
          >
            Services
          </a>
        </li>
        <li
          class="footer__list-item footer__list-item--desktop"
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
            Terms & Conditions
          </a>
        </li>
        <li
          class="footer__list-item footer__list-item--mobile"
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
            T&C's
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/faqs"
          >
            FAQ's
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/get-in-touch"
          >
            Contact
          </a>
        </li>
      </ul>
      <hr
        class="footer__line-break"
      />
      <div
        class="footer__secondary-links-container"
      >
        <div
          class="footer__copyright-text"
        >
          © 2022 The Positive Pup. All rights reserved.
        </div>
        <div
          class="footer__social-links"
        >
          <a
            href="https://www.facebook.com/thepositivepupdogtraining"
            target="blank"
          >
            <img
              alt="link to facebook"
              src="facebook-logo.svg"
            />
          </a>
          <a
            href="https://www.instagram.com/the_positivepup/"
            target="blank"
          >
            <img
              alt="link to instagram"
              src="instagram-logo.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
